src/arrayMethodJoin.js

```diff
@@ -4,9 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(separator = ',') {
     // write code here
+    let stringJoin = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const THIS_ITEM = this[i];
```

Comment:
it's not type of variable which is used in many places around your code (it's not global), it's just a part of this function, so it shouldn't be written in uppercase, just camelCase